cli/src/nonce.rs

```diff
@@ -363,12 +363,15 @@ pub fn process_create_nonce_account(
         (&nonce_account_pubkey, "nonce_account_pubkey".to_string()),
     )?;
 
-    if rpc_client.get_account(&nonce_account_pubkey).is_ok() {
-        return Err(CliError::BadParameter(format!(
-            "Unable to create nonce account. Nonce account already exists: {}",
-            nonce_account_pubkey,
-        ))
-        .into());
+    if let Ok(nonce_account) = rpc_client.get_account(&nonce_account_pubkey) {
+        let err_msg = if nonce_account.owner == system_program::id()
+            && State::<NonceState>::state(&nonce_account).is_ok()
+        {
+            "nonce account already exists"
```

Comment:
(This comment also for stake, storage, and vote below)
Can you please capitalize all of the err_msg, and also include each `_account_pubkey` in some fashion? (This has been helpful to diagnose if the user is simply passing the wrong key file.)
Perhaps something like "Nonce account {PUBKEY} already exists", "Account {PUBKEY} already exists and is not a nonce account"

Reply:
Done!